vulnerable_people_form/integrations/persistence.py

```diff
@@ -0,0 +1,203 @@
+import boto3
+
+from flask import current_app
+
+
+def get_rds_data_client(app=current_app):
+    return boto3.client(
+        "rds-data", endpoint_url=app.config.get("LOCAL_AWS_ENDPOINT_URL")
+    )
+
+
+def get_rds_client(app=current_app):
+    return boto3.client("rds", endpoint_url=app.config.get("LOCAL_AWS_ENDPOINT_URL"))
+
+
+def get_secretsmanager_client(app=current_app):
+    return boto3.client(
+        "secretsmanager", endpoint_url=app.config.get("LOCAL_AWS_ENDPOINT_URL")
+    )
+
+
+def _find_database_arn(app):
+    rds = get_rds_client(app=app)
+    clusters = rds.describe_db_clusters()
```

Comment:
this should be    ` clusters = rds.describe_db_clusters()['DBClusters']` as its the clusters information needed

Reply:
Thanks! 🙂